Makefile

```diff
@@ -30,8 +30,13 @@ endif
 EVE_SNAPSHOT_VERSION=0.0.0
 # which language bindings to generate for EVE API
 PROTO_LANGS=go python
-# Use 'make HV=acrn|xen|kvm' to build ACRN images (AMD64 only), Xen or KVM
+# Use 'make HV=acrn|xen|kvm|kubevirt' to build ACRN images (AMD64 only), Xen, Kubevirt or KVM
 HV=$(HV_DEFAULT)
+
+ifeq ($(HV),kubevirt)
```

Comment:
Line 761 does explicit check if HV = HV_DEFAULT.

Reply:
If you look on the line above ([749](https://github.com/lf-edge/eve/blob/master/Makefile#L749) on master)  you'll see that linuxkit is called anyways, the `if`  only checks for existence of EVE_REL and call the linuxkit with EVE_REL as release in case HV matches HV_DEFAULT. You shouldn't change HV_DEFAULT, it's suppose to be set only once....

Reply:
ok will update the commit